python/mxnet/optimizer.py

```diff
@@ -534,6 +535,66 @@ def update_multi_precision(self, index, weight, grad, state):
         self._update_impl(index, weight, grad, state,
                           multi_precision=use_multi_precision)
 
+@register
+class Signum(Optimizer):
+    """The Signum optimizer that takes the sign of gradient or momentum.
+
+    The optimizer updates the weight by:
+
+        rescaled_grad = rescale_grad * clip(grad, clip_gradient) + wd * weight
+        state = momentum * state + (1-momentum)*rescaled_grad
+        weight = (1 - lr * wd_lh) * weight - lr * sign(state)
```

Comment:
what's wd_lh? Is it from the original paper?

Reply:
It is an alternative weight decay. See the descriptions.

Reply:
Since `wd_lh` is new, I suggest put a reference link to the original paper by Loshchilov and Frank Hutter in the documentation